File: Trips.Backend/Trips.Core/DTO/Trip/TripTableDto.cs
using Trips.Core.Attributes.Searchable;

namespace Trips.Core.DTO.Trip;

public class TripTableDto
{
    [Searchable]
    public int Id { get; set; }
    [Searchable]
    public int DriverId { get; set; }    
    [Searchable]
    public string? Pickup { get; set; }
    [Searchable]
    public string? Dropoff { get; set; }
}

File: Trips.Frontend/Trips.Frontend/src/model/trip/tripTableDto.ts
import { Guid } from "guid-typescript";

export class TripTableDto {
    public Id! : Guid;
    public DriverId? : number;
    public Pickup? : string;
    public Dropoff? : string;

    public constructor(init?:Partial<TripTableDto>) {
        Object.assign(this, init);
    }
}